Content.Server/Chat/Commands/MeCommand.cs

```diff
@@ -50,7 +52,15 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 return;
             }
 
-            chat.EntityMe(mindComponent.OwnedEntity.Value, action);
+            var isEmote = _sanitizer.TrySanitizeOutSmilies(action, mindComponent.OwnedEntity.Value, out var sanitized, out var emoteStr);
+
+            if (sanitized.Length != 0)
+            {
+                chat.EntityMe(mindComponent.OwnedEntity.Value, sanitized);
+            }
+
+            if (isEmote)
+                chat.EntityMe(mindComponent.OwnedEntity.Value, emoteStr!);
```

Comment:
This should be done in chatmanager IMO like the other entity methods do.

Reply:
I agree with @metalgearsloth, instead of sending directly to EntityMe, modify TrySpeak to also handle emotes so its all done in one uniform way. If you want some help, let me know. I've been doing a lot inside the chat manager lately.

Reply:
Moved it to ChatManager.TryEmote, I didn't merge it with TrySpeak as TrySpeak also handles whispering and deadchat which TryEmote doesn't need to, so I thought it might be cleaner this way.